src/sentry/integrations/vercel/client.py

```diff
@@ -39,8 +43,22 @@ def get_user(self):
         return self.get_cached(self.USER_URL)["user"]
 
     def get_projects(self):
-        # TODO: we will need pagination since we are limited to 20
-        return self.get(self.PROJECTS_URL)["projects"]
+        limit = 20
+        params = {"limit": limit}
+        projects = []
+        # no one should have more than 200 projects
+        for i in range(10):
+            resp = self.get(self.PROJECTS_URL, params=params)
+            projects += resp["projects"]
```

Comment:
I'm assuming no projects returns `{"projects": []}` based on the tests you added

Reply:
@MeredithAnya Correct